File: an2_sem4/MPP/Homework/sapt10/cs/FestivalSellpoint/FestivalSellpoint.Network/ObjectProtocol/LoginAngajatRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FestivalSellpoint.Network.ObjectProtocol
{
    [Serializable]
    public class LoginAngajatRequest : IRequest
    {
        public string Username { get; }
        public string Password { get; }

        public LoginAngajatRequest(string username, string password)
        {
            Username = username;
            Password = password;
        }
    }
}

File: an2_sem4/MPP/Homework/sapt10/java/network/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/network/objectprotocol/LoginAngajatRequest.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.objectprotocol;

public class LoginAngajatRequest implements IRequest {
    private String username;
    private String password;

    public LoginAngajatRequest() {
    }

    public String getPassword() {
        return password;
    }

    public String getUsername() {
        return username;
    }

    public LoginAngajatRequest(String username, String password) {
        this.username = username;
        this.password = password;
    }
}
